_data-prepper/core-apis.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: default
+title: Core APIs
+nav_order: 2
+---
+
+# Core APIs
+
+All Data Prepper instances expose a server with some control APIs. By default, this server runs on port 4900. Some plugins, especially Source plugins may expose other servers. These will be on different ports and their configurations are independent of the core API. For example, to shut down Data Prepper, you can run:
```

Comment:
```suggestion
All Data Prepper instances expose a server with some control APIs. By default, this server runs on port 4900. Some plugins, especially Source plugins, may expose other servers. These will be on different ports, and their configurations are independent of the core API. For example, to shut down Data Prepper, you can run:
```

Reply:
Confirm the capitalization of "Source". Expand the last sentence to read "...you can run the following [noun]:"